src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptyIsFalse.java

```diff
@@ -0,0 +1,230 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
@romani @pbludov I am not sure, should we include such headers in newly created input files?

Reply:
@MahmoudAbdelazim , please remove such header
and remove whole comment "Contains simple mistakes: ......"

Reply:
@romani @timurt Done.